### ISSUE TITLE
SOR - Add support for LBPs

### ISSUE THREAD

**Issue by brunoguerios** [balancer/backend#1151]
Make sure this pool on gnosis is being picked up by the SOR: https://balancer.fi/pools/gnosis/v2/0x373B347BC87998B151A5E9B6BB6CA692B766648A0002000000000000000000D7

**Comment by brunoguerios** [comment]
Franz implemented this by handling LBPs as Weigthed pools.
I'm still checking with Daniel if there's need to add a final tweak, but likely not.

**Comment by brunoguerios** [comment]
The expectation is that LBPs can be handled as Weighted pools from the SOR perspective, but it would be nice to have a test scenario that would check if an LBP with no trades is properly picked up.

The assumption is that even without trades and NL defaulting to zero, SOR would make it a low priority, but not filter out the pool/path. This should be ok, because normally and LBP is a token launch and no other higher liquidity pool will have that token available for swapping.

In case no trades mean the pool is filtered out from path finding, then we should adapt the code to make sure it's included.

**Comment by franzns** [comment]
This is done?

**Comment by brunoguerios** [comment]
No, I didn't add the extra test and it's kinda low priority at the moment. Should take a while until I get to this.

**Comment by brunoguerios** [comment]
@franzns - the assumption here is that normalizedLiquidity computation is done after every swap event, so we'd need a swap on the LBP for its tokenPairData to be filled out in the DB, but that's not the only moment this sync is called, right?
Maybe a better question: when is `fetchTokenPairData` called other than after every swap event?
If we do call it on a cron job or when the pool is first picked up by subgraph, we don't actually need the extra test and we can close this issue.
